docs/pages/server-access/guides/jetbrains-sftp.mdx

```diff
@@ -65,6 +65,13 @@ $ ssh user@[node name].[cluster name]
 </Details>
 
 <Admonition type="note">
+  Include the port number for OpenSSH servers, by default `22`, or you can experience an error.
+  See the [OpenSSH guide](./openssh.mdx) for more information.
+  Example connecting to a OpenSSH server:
+  ```code
+  $ ssh -p 22 user@[node name].[cluster name]
+  ```
+  
   Teleport's certificates expire fairly quickly, after which SSH
   attempts will fail with an error like the following:
 
```

Comment:
Because this note now contains two separate topics, I suggest splitting them up. In this case, I think the original information in the note may be better presented at the end of Step 3, after they've begun using the setup and would begin experiencing certificate expiration.